package.json

```diff
@@ -21,11 +21,17 @@
   "devDependencies": {
     "async": "^0.9.0",
     "babel-core": "5.6.18",
```

Comment:
This is a pretty old version of `babel-core`.  We should probably use the most recent version of the `6.x` line.

Reply:
Changing to "babel-core": "^6", is that right? I notice wonder-blocks using 7

Reply:
webapp uses 6 right now.  I think we'll want to migrate everything to 7 eventually but there were some growing pains moving from 6 to 7 in wonder-blocks that we probably want to avoid at the moment.